### PR TITLE
Refactor: move feedback viewing functionality for PPs to Appointments Controller

### PR DESCRIPTION
**Pull request by Gweaton**
## What does this pull request do?

Moves functionality for viewing Supplier Assessment and Action Plan session assessments as a PP to the new Apppointments Controller.

## What is the intent behind these changes?

So we can reduce the size of the Probation Practitioner Referrals Controller as it's fairly bloated at the moment, and we'd like smaller, more focused controllers.

### CODE REVIEW

File: server/routes/appointments/appointmentsController.ts
Comment (andrewburrowsmoj) [inline server/routes/appointments/appointmentsController.ts:776]: Seems that there's no harm having this check on the `viewPostSessionFeedbackAsServiceProvider` so can be moved into the private method `viewSubmittedPostSessionFeedback`?
If you agree that also perhaps have a single method like with `viewSupplierAssessmentFeedback`?

**Review by andrewburrowsmoj — APPROVED** [review]
LGTM - I made a small comment; if you agree with it.

File: server/routes/appointments/appointmentsController.ts
Comment (Gweaton) [reply server/routes/appointments/appointmentsController.ts:776]: I did initially try to re-use the function but I think my brain wasn't working too well yesterday. I've had another go and think it makes more sense now - I know you've already approved, but I don't suppose you'd mind taking another look as it'd be nice to get a second pair of eyes on the approach.

**Pull request merged**
